src/tests/system/tests/test_passkey.py

```diff
@@ -740,3 +740,151 @@ def test_passkey__su_with_12_mappings(
+        username="user1",
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.ipa",
+        command="klist",
+        auth_method="PasskeyAuthenticationUseCases.PASSKEY_NO_PIN_NO_PROMPTS",
+    )
+
+    ipa.host.conn.run("ipa passkeyconfig-mod --require-user-verification=True", raise_on_error=False)
+
+    assert rc == 0, "Authentication failed"
+    assert "Ticket cache" in output, "Failed to get the TGT"
+    assert not (
+        (
+            "No Kerberos TGT granted as the server does not support this method. "
+            "Your single-sign on(SSO) experience will be affected"
+        )
+        in output
+    ), "Get the console message about TGT"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.builtwith(client="passkey", ipa="passkey")
+@pytest.mark.ticket(gh=6931)
+def test_passkey__prompt_options(
+    client: Client, ipa: IPA, moduledatadir: str, testdatadir: str, umockdev_ipaotpd_update
+):
+    """
+    :title: Check authentication of user with updated prompting options
+    :setup:
+        1. Add a user in the server with passkey mappings
+        2. Add the prompting options to sssd.conf file
+        3. Setup SSSD client with FIDO and umockdev, start SSSD service
+    :steps:
+        1. Check authentication of the user
+        2. Check the updated prompt options
+    :expectedresults:
+        1. User authenticates successfully
+        2. Got the update prompt options
+    :customerscenario: False
+    """
+    with open(f"{testdatadir}/passkey-mapping.ipa") as f:
+        ipa.user("user1").add(user_auth_type="passkey").passkey_add(f.read().strip())
+
+    client.sssd.section("prompting/passkey")["interactive"] = "True"
+    client.sssd.section("prompting/passkey")["interactive_prompt"] = "Please, insert the passkey and press enter"
+    client.sssd.section("prompting/passkey")["touch"] = "True"
+    client.sssd.section("prompting/passkey")["touch_prompt"] = "Can you touch the passkey"
+    client.sssd.start(service_user="root")
+
+    rc, _, output, _ = client.auth.su.passkey_with_output(
+        username="user1",
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.ipa",
+        pin=123456,
+        interactive_prompt="Please, insert the passkey and press enter",
+        touch_prompt="Can you touch the passkey",
+        command="klist",
+        auth_method="PasskeyAuthenticationUseCases.PASSKEY_PIN_AND_PROMPTS",
+    )
+
+    assert rc == 0, "Authentication failed"
+    assert "Ticket cache" in output, "Failed to get the TGT"
+    assert (
+        not (
+            "No Kerberos TGT granted as the server does not support this method."
+            "Your single-sign on(SSO) experience will be affected"
+        )
+        in output
+    ), "Get the console message about TGT"
```

Comment:
A slightly more detailed message would make this more clear.  Maybe something like:

```
Got the console message about No Kerberos TGT granted
```

Reply:
updated